src/CMakeLists.txt

```diff
@@ -677,6 +677,7 @@ set (H5VL_HDRS
     ${HDF5_SRC_DIR}/H5VLconnector_passthru.h
     ${HDF5_SRC_DIR}/H5VLnative.h
     ${HDF5_SRC_DIR}/H5VLpassthru.h
+    ${HDF5_SRC_DIR}/H5VLpassthru_private.h
```

Comment:
This section is only for public headers to be installed, though I don't think we need the H5VLpassthru_private.h header.